scripts/prepare_deps.sh

```diff
@@ -5,13 +5,21 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 source $DIR/_common.sh
 
-# Build the node_modules Docker layer and store it in the shared cache so it can be reused later
+# Build the node_modules as well as the base image for the final outputs and store it in the shared cache so it can be reused later
 
 mkdir -p $PROJECT_ROOT/cache
-docker buildx create --driver docker-container --use
-exec docker buildx build \
+docker buildx create --driver docker-container --use || true
+
+docker buildx build \
+  --platform=linux/amd64 \
+  --cache-to=type=local,dest=$PROJECT_ROOT/cache_output \
+  -f ${DIR}/Dockerfile \
+  --target=output-base \
+  $PROJECT_ROOT
+
+docker buildx build \
```

Comment:
wouldn't it make more sense to build deps before we build output-base?

Reply:
Does not really matter, was switching the order just to see if it will affect the cache. Seems not.

Reply:
fixed